### ISSUE TITLE
Adding prices on the bottom right of extensions in the addons page

### ISSUE THREAD

**Issue by V3NCO** [BlueprintFramework/web#9]
My idea is to add prices on the [blueprint.zip](https://blueprint.zip) website and maybe add it as a filter ? I'm not able to do html/css/anything so i can't make a pull request for that, but it's a little suggestion.

**Comment by prplwtf** [comment]
Has been an idea before, but it's a bit difficult due to price changes. The entire browse page is updated manually, so we'd have to constantly update prices, as those change more frequently compared to product thumbnails.

I'll keep this open for the time being so anyone with additional suggestions on how to achieve this can tell us through this issue.

**Comment by AucuneInspiexe** [comment]
Maybe Built by bit or/and sourcexchange has a public API ? That could help get the prices in real time

**Comment by Jelcoo** [comment]
https://www.postman.com/sourcexchange/workspace/sourcexchange/overview
The api docs for sourcexchange. It is not much, but create a ticket in our discord (https://discord.com/invite/XgtSDWwGXx) if you need a new endpoint.

**Comment by prplwtf** [comment]
I have BBB Ultimate, so the API pricing is not a problem.
For the static website part, stuff can potentially be routed through my BlueprintAPI server that's also used for the telemtry endpoint.

**Comment by itsvic-dev** [comment]
That could work yeah, just make sure that you're restricting what routes you can actually access through BlueprintAPI.

**Comment by Jelcoo** [comment]
Maybe its a good idea to make the whole list more dynamic by using the api.

**Comment by prplwtf** [comment]
I think moving over the **entire** list to BlueprintAPI, then just making that fetchable. Would allow to fetch (and cache) prices on the server-side.

Obviously the static repository will still be used as fallback.

**Comment by AucuneInspiexe** [comment]
Yeah that would make it more dynamic, sourceXchange even has a blueprint category, if theres an endpoint for that... Even though   Builtbybit doesn't

**Comment by prplwtf** [comment]
I'll contact them if needed. For now though, I'll keep this open, as there are a couple other things that are having my priorities at the moment.

**Comment by AucuneInspiexe** [comment]
Okay

**Comment by 0x7d8** [comment]
@prplwtf bump

**Comment by 0x7d8** [comment]
@prplwtf this can be relooked at with #31, the api now provides the prices, only a ui is missing
